Documentation/community/governance.rst

```diff
@@ -7,9 +7,7 @@
 .. _governance:
 
 Governance
-----------
+==========
 
-.. toctree::
-   :maxdepth: 3
-
-   commit_access
+Governance documentation can be found in the Cilium Community repository:
+  https://github.com/cilium/community/blob/main/GOVERNANCE.md
```

Comment:
This renders ok 
![Screenshot from 2024-04-04 09-53-40](https://github.com/cilium/cilium/assets/3199463/f6565999-0a37-4a2d-b62f-91f60876703d)

although I'd write it like
```
Governance
----------

Governance documentation can be found in the `Cilium Community repository <https://github.com/cilium/community/blob/main/GOVERNANCE.md>`__.
```

Reply:
Thanks for your suggestion! I've adjusted it.